### ISSUE TITLE
Alternative text missing from Division > Collection Strengths section

### ISSUE THREAD

**Issue by axamei** [pulibrary/special_collections#526]
e.g., https://rbsc.princeton.edu/divisions/public-policy-papers

**Comment by sandbergja** [comment]
These images now have an alt text if it is supplied, and an empty alt attribute if not.  Closing this ticket as completed.
